drivers/okhttp/src/main/java/org/jclouds/http/okhttp/OkHttpCommandExecutorService.java

```
@@ -178,6 +179,11 @@ protected HttpResponse invoke(Request nativeRequest) throws IOException, Interru
    }
 
    @Override
+   protected ListenableFuture<HttpResponse> invokeAsync(final Request nativeRequest) {
+      return null;
```

Lets throw unsupported exceptions here too